### PR TITLE
Add and remove monitoring metrics

### PR DESCRIPTION
**Pull request by Barthelemy**
Keep only 4 metrics : 
- qc_objects_received
- qc_checks_executed 
- qc_mo_stored
- qc_qo_stored

They all have their rate published.

### CODE REVIEW

File: Framework/include/QualityControl/CheckRunner.h [hunk 1]
```diff
@@ -193,9 +194,12 @@ class CheckRunner : public framework::Task
 
   // monitoring
   std::shared_ptr<o2::monitoring::Monitoring> mCollector;
-  std::chrono::system_clock::time_point startFirstObject;
-  std::chrono::system_clock::time_point endLastObject;
+  //  std::chrono::system_clock::time_point startFirstObject;
+  //  std::chrono::system_clock::time_point endLastObject;
   int mTotalNumberHistosReceived;
```
Comment (knopers8) [inline Framework/include/QualityControl/CheckRunner.h:199]: Could you take an opportunity to change it to `mTotalNumberMOReceived` for example? Histos are a bit misleading.

File: Framework/src/CheckRunner.cxx [hunk 2]
```diff
@@ -254,10 +250,12 @@ void CheckRunner::run(framework::ProcessingContext& ctx)
...
 
   // monitoring
-  endLastObject = system_clock::now();
   if (timer.isTimeout()) {
     timer.reset(1000000); // 10 s.
-    mCollector->send({ mTotalNumberHistosReceived, "objects" }, o2::monitoring::DerivedMetricMode::RATE);
+    mCollector->send({ mTotalNumberHistosReceived, "qc_objects_received" }, DerivedMetricMode::RATE);
+    mCollector->send({ mTotalNumberCheckExecuted, "qc_checks_executed" }, DerivedMetricMode::RATE);
+    mCollector->send({ mTotalNumberQOStored, "qc_qo_stored" }, DerivedMetricMode::RATE);
+    mCollector->send({ mTotalNumberMOStored, "qc_mo_stored" }, DerivedMetricMode::RATE);
```
Comment (knopers8) [inline Framework/src/CheckRunner.cxx:258]: Perhaps we should stick either to `objects` or `mo`, since we mean the same. In TaskRunner we use `objects`.

Comment (Barthelemy) [reply Framework/src/CheckRunner.cxx:258]: actually I had a commit that I forgot to push that did exactly that.

Comment (Barthelemy) [reply Framework/src/CheckRunner.cxx:258]: wait, I meant another one, here I wanted to differentiate MOs from QO stored. I could merge them and then I would call them objects.

File: Framework/include/QualityControl/CheckRunner.h (hunk 1 above)
Comment (Barthelemy) [reply Framework/include/QualityControl/CheckRunner.h:199]: totally agree

File: Framework/src/CheckRunner.cxx (hunk 2 above)
Comment (knopers8) [reply Framework/src/CheckRunner.cxx]: Differentiating MOs and QOs makes sense completely. I was more reluctant to use `objects` and `mo` interchangeably. However, as I think of it now, we will allow not only MOs here, but also other kinds of objects might be injected here, which won't be anyway stored. So the distinction is actually justified here.

**Comment by Barthelemy** [comment]
@MichaelLettrich I think that there is still a problem with o2-dataflow

**Comment by Barthelemy** [comment]
there is a stuck travis but the other one is ok.

**Pull request merged**